test/itkAttenuationImageFilterTest.cxx

```diff
@@ -126,5 +128,42 @@ itkAttenuationImageFilterTest(int argc, char * argv[])
     itk::WriteImage(attenuationFilter->GetOutputMaskImage(), outputMaskImagePath);
   }
 
+  // discover mask's inside value
+  using MaxType = itk::MinimumMaximumImageCalculator<MaskImageType>;
+  auto maxCalculator = MaxType::New();
+  maxCalculator->SetImage(attenuationFilter->GetOutputMaskImage());
+  maxCalculator->ComputeMaximum();
+  LabelType maxMaskValue = maxCalculator->GetMaximum();
+  if (maxMaskValue == 0)
+  {
+    std::cerr << "The mask has been completely eroded away!" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // now boil it down to a single attenuation value
+  using HistogramFilterType = itk::Statistics::MaskedImageToHistogramFilter<OutputImageType, MaskImageType>;
+  auto histogramFilter = HistogramFilterType::New();
+  histogramFilter->SetInput(attenuationFilter->GetOutput());
+  histogramFilter->SetMaskImage(attenuationFilter->GetOutputMaskImage());
+  histogramFilter->SetMarginalScale(10);
+
+  HistogramFilterType::HistogramSizeType histogramSize{ 1 };
+  histogramSize[0] = 1e5;
+  histogramFilter->SetHistogramSize(histogramSize);
+
+  histogramFilter->SetMaskValue(maxMaskValue); // this is most likely the only label present
+  histogramFilter->Update();
+  auto histogram = histogramFilter->GetOutput();
+
+  // we will use median as a robust estimate of the mean
+  auto median = histogram->Quantile(0, 0.50);
```

Comment:
Would it be reasonable to expose the median ground truth value as a command line argument so that we can automatically check for accuracy?

Reply:
It would. But I plan to modify this filter soon, which mean baseline values will change. We could do that afterwards.